processor/schemaprocessor/internal/fixture/parallel.go

```diff
@@ -0,0 +1,56 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Could this file not include the following in the header to avoid running when `-race` is not enabled?

```
//go:build race
// +build race
```

Reply:
So the intent with this is that I wanted it to be clear that when running tests without `-race` that a test was skipped, rather than a test was missing.

Reply:
With @tigrannajaryan comment , I've updated the godoc to explain this.